models/eprop_readout_bsshslm_2020.cpp

```diff
@@ -294,6 +293,8 @@ eprop_readout_bsshslm_2020::update( Time const& origin, const long from, const l
 
     error_signal_buffer[ lag ] = S_.error_signal_;
 
+    emplace_new_eprop_history_entry( t );
```

Comment:
See comment elsewhere on `emplace`.

Reply:
Fixed in [5356e64](https://github.com/nest/nest-simulator/pull/3207/commits/5356e6478d2f0c2410723d8ce5b146036fa418a7)